src/com/google/enterprise/adaptor/prebuilt/DateFilter.java

```diff
@@ -39,15 +43,19 @@
  * {@code key} The name of the metadata or param property whose date value
  * determines whether the document will be skipped or not.
  * <p>
- * {@code format} The {@link SimpleDateFormat} used to parse the document's
+ * {@code format} The {@link DateFormat} used to parse the document's
  * date values. If no {@code format} is specified, a lenient ISO8601 format
- * ("yyyy-MM-dd") is used.
+ * ("yyyy-MM-dd") is used. If format is "millis", then the dates
+ * are parsed as if they were milliseconds since the epoch
+ * (January 1, 1970, 00:00:00 GMT). Otherwise, the format must be parsable
+ * by {@link SimpleDateFormat}.
  * <p>
  * {@code date} The cut-off for the date value. Document's whose date
  * value is before the configued {@code date} will not be indexed.
  * The configured {@code date} must be parsable by the configured date
- * {@code format}. Only one of {@code date} or {@code days} configuration
- * may be specified.
+ * {@code format}, unless {@code format} is "millis", in which
+ * case the date must be specified in ISO8601 format. Only one of
+ * {@code date} or {@code days} configuration may be specified.
```

Comment:
Please insert example configs. One with millis that  would work with last_modified param.